plugins/auth/index.js

```diff
@@ -140,30 +140,58 @@ exports.register = (server, options, next) => {
                 accessTokenName: 'api_token',
                 allowCookieToken: false,
                 allowQueryToken: true,
-                validateFunc: function _validateFunc(token, cb) {
+                validateFunc: function _validateFunc(tokenValue, cb) {
                     // Token is an API token
                     // using function syntax makes 'this' the request
                     const request = this;
-                    const factory = request.server.app.userFactory;
-
-                    return factory.get({ accessToken: token })
-                        .then((user) => {
-                            if (!user) {
-                                return cb(null, false, {});
+                    const tokenFactory = request.server.app.tokenFactory;
+                    const userFactory = request.server.app.userFactory;
+                    const pipelineFactory = request.server.app.pipelineFactory;
+
+                    tokenFactory.get({ value: tokenValue })
```

Comment:
Can we have `return` before this? just to be consistent

Reply:
Okay, I'll fix it.